framework/tlvf/yaml/tlvf/WSC/WSC_Attributes.yaml

```diff
@@ -395,3 +407,25 @@ cWscAttrEncryptedSettings:
   encrypted_settings:
     _type: char
     _length: []
+
+cWscVendorExt:
```

Comment:
Since the WFA ID is hardcoded here, it would be better to name it `cWscVendorExtWfa`.